src/main/java/io/hhplus/tdd/point/PointService.java

```diff
@@ -6,13 +6,19 @@
 import org.springframework.stereotype.Service;
 
 import java.util.List;
+import java.util.concurrent.locks.ReentrantLock;
 
 @Service
 @RequiredArgsConstructor
 public class PointService {
     private final UserPointTable userPointTable;
     private final PointHistoryTable pointHistoryTable;
 
+    private final java.util.concurrent.ConcurrentHashMap<Long, java.util.concurrent.locks.ReentrantLock> lockMap = new java.util.concurrent.ConcurrentHashMap<>();
+    private java.util.concurrent.locks.ReentrantLock lockFor(long userId) {
+        return lockMap.computeIfAbsent(userId, k -> new java.util.concurrent.locks.ReentrantLock());
+    }
+
```

Comment:
```suggestion
    private final ConcurrentHashMap<Long, ReentrantLock> lockMap = new ConcurrentHashMap<>();
    private ReentrantLock lockFor(long userId) {
        return lockMap.computeIfAbsent(userId, k -> new ReentrantLock());
    }
```
import java.util.concurrent.ConcurrentHashMap 추가필요

Reply:
안녕하세요.
아무래도 C++을 주로 쓰시다보니 네임스페이스를 풀로 작성하는 습관이 있으신거같네요 ㅎㅎ
자바에서는 가독성을 위해서 import하여 쓰는것이 컨벤션이라 익숙해지시면 좋을거같아요.
저도 C++을 많이 썼었는데 반갑네용